dapr/proto/runtime/v1/dapr.proto

```diff
@@ -30,6 +30,12 @@ service Dapr {
   // Invokes a method on a remote Dapr app.
   rpc InvokeService(InvokeServiceRequest) returns (common.v1.InvokeResponse) {}
 
+  // Get health of a component
+  rpc GetComponentHealthAlpha1(ComponentHealthRequest) returns (ComponentHealthResponseItem) {}
+
+  // Get health of all registered components
+  rpc GetAllComponentsHealthAlpha1(AllComponentsHealthRequest) returns (AllComponentsHealthResponse) {}
```

Comment:
Can we do both in GetAllComponentsHealthAlpha1 and let the request filter which one(s) and default to all if empty?